### DISCUSSION TITLE
My stat counter isn't working.

### DISCUSSION THREAD

**Discussion by wednesday2024** [anuraghazra/github-readme-stats#2753] [Q&A] [answered]
The title describes my issue. I would say for the past 2 months now my stat counter hasn't been displayed correctly. My stats used to show up properly then one day it just started showing zeros and hasn't changed since. What am I doing wrong?

Here is the code that I have set for myself:

`![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=R5-3600&include_all_commits=true&count_private=true&show_icons=true&theme=transparent)`

Here is my GitHub profile for if anyone wants to see what I am seeing:

https://github.com/R5-3600

Thank you.

**Comment by rickstaa** [comment]
[![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=R5-3600)](https://github.com/anuraghazra/github-readme-stats)

@R5-3600 I checked your profile and the card seems to be correct. You can check the data we receive from GitHub in the [GraphQL explorer](https://docs.github.com/en/graphql/overview/explorer):

```graphql
query userInfo($login: String!) {
  user(login: $login) {
    name
    login
    contributionsCollection {
      totalCommitContributions
      restrictedContributionsCount
    }
    repositoriesContributedTo(first: 1, contributionTypes: [COMMIT, ISSUE, PULL_REQUEST, REPOSITORY]) {
        totalCount
      }
      pullRequests(first: 1) {
        totalCount
      }
      openIssues: issues(states: OPEN) {
        totalCount
      }
      closedIssues: issues(states: CLOSED) {
        totalCount
      }
      followers {
        totalCount
      }
  }
}
```

```graphql
{"login": "R5-3600"}
```

Please [deploy your own vercel instance](https://github.com/anuraghazra/github-readme-stats#deploy-on-your-own) if you want private statistics to show up.

**Reply by wednesday2024** [reply]
> [![Anurag's GitHub stats](https://camo.githubusercontent.com/cd63a1cd77abf03b1df7524b1e5ac502dcf2872528bb30e95276e1ee3da2eae1/68747470733a2f2f6769746875622d726561646d652d73746174732e76657263656c2e6170702f6170693f757365726e616d653d52352d33363030)](https://github.com/anuraghazra/github-readme-stats)
> 
> @R5-3600 I checked your profile and the card seems to be correct. You can check the data we receive from GitHub in the [GraphQL explorer](https://docs.github.com/en/graphql/overview/explorer):
> 
> ```graphql
> query userInfo($login: String!) {
>   user(login: $login) {
>     name
>     login
>     contributionsCollection {
>       totalCommitContributions
>       restrictedContributionsCount
>     }
>     repositoriesContributedTo(first: 1, contributionTypes: [COMMIT, ISSUE, PULL_REQUEST, REPOSITORY]) {
>         totalCount
>       }
>       pullRequests(first: 1) {
>         totalCount
>       }
>       openIssues: issues(states: OPEN) {
>         totalCount
>       }
>       closedIssues: issues(states: CLOSED) {
>         totalCount
>       }
>       followers {
>         totalCount
>       }
>   }
> }
> ```
> 
> ```graphql
> {"login": "R5-3600"}
> ```
> 
> Please [deploy your own vercel instance](https://github.com/anuraghazra/github-readme-stats#deploy-on-your-own) if you want private statistics to show up.

I deployed my own version on Vercel but now it states this: 
```Something went wrong! file an issue at https://tiny.one/readme-stats
Maximum retries exceeded Please add an env variable called PAT_1 with your github token in vercel
```

**Reply by rickstaa** [reply, marked as answer]
Please follow step 6-11 from the step-by-step guide. The video is not up-to-date any more 😅.

**Reply by wednesday2024** [reply]
> Please follow step 6-11 from the step-by-step guide. The video is not up-to-date any more 😅.

That worked. Thank you.
